exdb/tests/browser_tests.py

```diff
@@ -217,7 +217,11 @@ def test_load(self):
 
 
 class HallStaffDashboardBrowserTest(DefaultLiveServerTestCase):
-
+    """
+    TODO: Once we have a login page put this back in...
```

Comment:
This "TODO" should be in a comment so that the linter will catch it. I would hate for us to have a test accidentally get swept under the rug because of this. We should either fix this or have it fail loudly. If we are fine with having a test fail for a bit on master we can still merge this even if it is failing.